components/lib/api.js

```diff
@@ -10,21 +11,29 @@ export const apiEndpoints = {
   COUNTRIES_LIST: '/api/_/countries',
   // Submissions
   SUBMISSION_LIST: '/api/_/url-submission/test-list',
-  SUBMISSION_ADD: '/api/v1/url-submission/add-url',
   SUBMISSION_UPDATE: '/api/v1/url-submission/update-url',
-  SUBMISSION_STATE: '/api/v1/url-submission/state',
-  SUBMISSION_CHANGES: '/api/v1/url-submission/changes',
   SUBMISSION_SUBMIT: '/api/v1/url-submission/submit'
 }
 
+const token = () => {
```

Comment:
Perhaps to prevent confusion with the `token` variable inside of `loginUser` this could be called `bearerToken` or `getBearerToken`?